pwa-devdocs/src/tutorials/pwa-studio-fundamentals/project-structure/index.md

```diff
@@ -0,0 +1,139 @@
+Here, you can add your custom reducers and enhancers to Peregrine's global store.
+
+For more information about state management, see: [State management][]
+
+### Important files outside `src`
+
+#### `webpack.config.js`
+
+The `webpack.config.js` is a Webpack configuration file.
+It uses [`configureWebpack`][] from PWA Studio's buildpack to create a configuration object for Webpack.
+
+#### `upward.yml`
+
+The `upward.yml` file is an [UPWARD][] configuration file.
+It provides provides instructions for how an UPWARD server implementation should respond to a request.
+
+The `status`, `headers`, and `body` values defined in the default `upward.yml` file uses values from the `veniaResponse` object.
+This object is defined in the [venia-ui package's `upward.yml` file][].
+
+Edit this file if you want to change how your project's UPWARD server responds to requests.
+
+#### `template.html`
+
+The `template.html` file contains the template for the server-side rendered HTML code for your storefront's application shell.
+
+During the build process, the build script injects this file with dynamic data, such as project variables and imports for Webpack-generated Javascript bundles.
+The script saves this new file as `dist/index.html`.
+When a browser creates the initial request for your storefront application, the default `upward.yml` configuration tells the UPWARD server to respond with this index file.
+
+As the JavaScript bundles load in the browser, the React components hydrate the different visual elements on the page with separate network requests.
+
+#### `server.js`
+
+The `server.js` file is the script that starts the production or development server.
+The script loads the `NODE_ENV` variable from the environment or your project's `.env` file to determine which type of server to start.
+
+Use the following values for `NODE_ENV`:
+
+-   `production` - start a production server
+-   `test` - start a staging server that uses staging server environment configurations
+-   `development` - start a development server that uses the development server environment configurations
+
+For more information on loading environment variables, see: [Load environment file][]
+
+[modular components]: <{%link venia-pwa-concept/features/modular-components/index.md %}>
+[project setup]: <{%link tutorials/pwa-studio-fundamentals/index.md %}>
+[`configurewebpack`]: <{%link pwa-buildpack/reference/configure-webpack/index.md %}>
+[upward]: <{%link technologies/upward/index.md %}>
+[load environment file]: <{%link pwa-buildpack/reference/buildpack-cli/load-env/index.md %}#programmatic-api>
+[state management]: <{%link technologies/basic-concepts/state-management/index.md %}>
+
+[service worker features]: https://developers.google.com/web/fundamentals/primers/service-workers
+[jest]: https://jestjs.io/
+[venia-ui package's `upward.yml` file]: https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/upward.yml
```

Comment:
[HtmlWebpackPlugin]: https://webpack.js.org/plugins/html-webpack-plugin/

Reply:
is this a blank comment?

Reply:
What? Oh man. Edited. It was supposed to add a link to HtmlWebpackPlugin